src/Classes/Controllers/GetEventsController.php

```diff
@@ -51,6 +51,13 @@ public function __invoke(Request $request, Response $response, array $args)
                 'data' => $events
             ];
             $statusCode = 200;
+        } else {
+            $data = [
+                'success' => true,
+                'message' => 'There is no event',
+                'data' => []
+            ];
+            $statusCode = 200;
```

Comment:
Could be code 204
"204 No Content
    The server successfully processed the request and is not returning any content."